packages/communication-react/review/communication-react.api.md

```diff
@@ -1419,12 +1419,13 @@ export const MeetingComposite: (props: MeetingCompositeProps) => JSX.Element;
 export type MeetingCompositePage = 'accessDeniedTeamsMeeting' | 'configuration' | 'joinMeetingFailedDueToNoNetwork' | 'leftMeeting' | 'lobby' | 'meeting' | 'removedFromMeeting';
 
 // @beta
-export type MeetingCompositeProps = {
-    meetingAdapter: MeetingAdapter;
+export interface MeetingCompositeProps extends BaseCompositeProps<CallCompositeIcons & ChatCompositeIcons> {
```

Comment:
@PorterNan can you confirm we are leaving meeting composite out of the final api file?

Reply:
Confirmed with Nan, all good here